modules/microvm/virtualization/microvm/audiovm.nix

```diff
@@ -72,6 +72,17 @@ let
             ] ++ lib.optional config.ghaf.development.debug.tools.enable pkgs.alsa-utils;
           };
 
+          users.users."proxy-user-audio" = {
+            isNormalUser = true;
+            uid = config.ghaf.users.accounts.loginuid;
```

Comment:
why?

Reply:
As discussed, dbus needs user account with same uid for authorization